src/nft.ts

```diff
@@ -26,6 +26,11 @@ import {
 } from './types';
 
 export default class Nft extends AinftBase {
+  /**
+   * Add nfy symbol. You can add nft to reference in your factory activity.
+   * @param param0
```

Comment:
It appears that the object parameters are currently labeled as param0. 
How about clarifying object types instead of param0?

Reply:
Updated. Thanks!